src/hierarchies/probit_normal_hierarchy.hpp

```diff
@@ -0,0 +1,103 @@
+#ifndef BAYESMIX_HIERARCHIES_PROBIT_NORMAL_HIERARCHY_HPP_
+#define BAYESMIX_HIERARCHIES_PROBIT_NORMAL_HIERARCHY_HPP_
+
+#include <google/protobuf/stubs/casts.h>
+
+#include <Eigen/Dense>
+#include <cassert>
+#include <memory>
+
+#include "../../proto/cpp/hierarchy_prior.pb.h"
+#include "../../proto/cpp/marginal_state.pb.h"
+#include "base_hierarchy.hpp"
+
+class ProbitNormalHierarchy : public BaseHierarchy {
```

Comment:
Again, I just added a hierarchy class out of habit without much thought, since I was focused on the implementation of covariates on a general level into the library. I will fix this as well